### DISCUSSION TITLE
[asdf] add support for istioctl and cookiecutter

### DISCUSSION THREAD

**Discussion by kamilaz** [renovatebot/renovate#30887] [Suggest an Idea]
### Tell us more.

Add support for istioctl and cookiecutter in asdf manager:
https://github.com/virtualstaticvoid/asdf-istioctl
https://github.com/shawon-crosen/asdf-cookiecutter

**Comment by viceice** [comment]
feel free to open a PR to add

https://github.com/renovatebot/renovate/blob/ff09ed56117d4d4736aa814a152799d45a1daadf/lib/modules/manager/asdf/upgradeable-tooling.ts#L42

**Comment by kamilaz** [comment]
Added https://github.com/renovatebot/renovate/pull/30890
